nebula/ui/components/forms/MZRadioButton.qml

```diff
@@ -26,8 +27,8 @@ RadioDelegate {
     onFocusChanged: {
```

Comment:
Should we change this one to also be `onActiveFocusChanged` ?

Reply:
Shouldn't make a difference here, but let's do it for consistency!